src/FileManager.php

```diff
@@ -203,7 +203,7 @@ public function extractRuleBlock(string $marker) : array {
      *
      * @param string $needle
      *
-     * @return int|bool
+     * @return bool|int|string
```

Comment:
Union types are only supported in PHP 8 ;-)

https://stackoverflow.com/a/37033355/753676

Reply:
Hi there,
it's correct that PHP 8 added support for union types. You are commenting on PHPDoc, though. This is treated like a comment by PHP.

This means the PHP code will run using both PHP 7.4 and 8.0. An IDE like PhpStorm can use the PHPDOC to offer hints to the developers.

Please note I removed the return type of the function in the PHP code due to the compatibility with PHP 7.4.

Reply:
Yes, I'm aware of that. Just wanted to note that the syntax was not compatible, which was fixed in https://github.com/igorhrcek/wp-cli-secure-command/commit/04d9265c0f18ca5d1e9ba2e61ce6afddc1e874f1 =)